src/cli/commands/publish.js

```diff
@@ -44,6 +44,7 @@ async function publish(config: Config, pkg: any, flags: Object, dir: string): Pr
   await config.executeLifecycleScript('prepublish');
   await config.executeLifecycleScript('prepare');
   await config.executeLifecycleScript('prepublishOnly');
+  await config.executeLifecycleScript('prepack');
```

Comment:
Shouldn't those lifecycles be part of the `pack` command rather than `publish`?

Reply:
They already are, but they are in pack's `run()` function which is not called directly by `publish`. I considered moving them into the commonly called `pack()` function, but `postpack` shouldn't be executed until the output stream is written, so it makes refactoring the scripts to be called in a common spot a bit more complicated. I took the easier route here.

Reply:
Shall we put that explanation in the code as a comment?